neuralop/models/uno.py

```diff
@@ -139,12 +146,25 @@ def __init__(self,
         self.preactivation = preactivation
         self._incremental_n_modes = incremental_n_modes
 
-        assert len(self.layer_configs) == n_layers
+        
+
+        self.layer_configs = []
+        for l in range(n_layers):
+            l_config = {}
+            l_config['out_channels'] = uno_out_channels[l]
+            l_config['n_modes'] = uno_n_modes[l]
+            l_config['res_scaling'] = uno_scalings[l]
+            self.layer_configs.append(l_config)
+        
+        if self.horizontal_skips_map is None:
+            self.horizontal_skips_map = {}
+            for i in range(n_layers//2,0,):
+                self.horizontal_skips_map[n_layers - i -1] = i
```

Comment:
I think we could just remove the horizontal skips map and just assume a regular UNO architecture. That's 99% of the usecases.

Reply:
It makes the forward call cleaner. Please note that **horizontal_skips_map** can be set to **None**, and then it fills up the **horizontal_skips_map** with a regular skip connection.